### PR TITLE
Reworded video to media

### PR DESCRIPTION
**Pull request by simondate**
The word video is used several times as an all encompassing word for what could also be an audio file. Should use the generic term `media` instead.

### CODE REVIEW

File: properties.schema [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "ariaRegion": {
       "type": "string",
       "required": true,
-      "default": "This is a media component which plays a video. Select the play / pause button to watch or listen.",
+      "default": "This is a media component which plays media. Select the play / pause button to watch or listen.",
```
Comment (moloko) [inline properties.schema]: Could you change to 'This is a component which plays media.'?

File: properties.schema [hunk 2]
```diff
@@ -143,7 +143,7 @@
       "title": "Set completion of Video on",
       "inputType": {"type": "Select", "options":["inview", "play", "ended"]},
       "validators": ["required"],
-      "help": "This tells Adapt when to set this video to complete"
+      "help": "This tells Adapt when to set this media to complete"
```
Comment (moloko) [inline properties.schema]: could you change to: 'Defines what media event should trigger completion of this component.'?

File: properties.schema [hunk 3]
```diff
@@ -143,7 +143,7 @@
       "title": "Set completion of Video on",
```
Comment (moloko) [inline properties.schema]: I'm thinking 'Completion trigger' might be a better title here?

File: properties.schema [hunk 4]
```diff
@@ -193,7 +193,7 @@
           "title": "Set Completion",
           "inputType": {"type": "Boolean", "options": [true, false]},
           "validators": [],
-          "help": "This tells Adapt to set this video to complete when the inline transcript is shown"
+          "help": "This tells Adapt to set this media to complete when the inline transcript is shown"
```
Comment (moloko) [inline properties.schema]: Could you change to 'Set to 'true' to have this component mark as completed when the inline transcript is shown.'

File: properties.schema [hunk 5]
```diff
@@ -193,7 +193,7 @@
           "title": "Set Completion",
```
Comment (moloko) [inline properties.schema]: I think 'Trigger completion?' would be a better title

Comment (simondate) [inline properties.schema]: With the question mark too?

**Pull request merged**